hawc/apps/myuser/api.py

```diff
@@ -25,4 +26,8 @@ class HawcValidateAuthToken(APIView):
     permission_classes = [IsAuthenticated]
 
     def get(self, request, format=None):
+        # also login for use of other views
```

Comment:
why is this needed? suggest removing

Reply:
authenticate/login functions aren't called automatically when visiting a page. The easiest path was adding them to this API endpoint. I tried doing this with a middleware but I couldn't get it to work